include/rmm/mr/device/per_device_resource.hpp

```diff
@@ -24,6 +24,8 @@
 #include <map>
 #include <mutex>
 
+#include <cuda/memory_resource>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Is this include necessary? Seems unused in this file?

Reply:
This pulls in `cuda::has_property`